modules/framework-cats/jvm/src/test/scala/junit/JUnitRunnerTests.scala

```diff
@@ -181,16 +181,14 @@ object JUnitRunnerTests extends SimpleIOSuite {
 
       val expected = List(
         TestSuiteStarted("weaver.junit.Meta$OnlyFailsOnCiEvenIfIgnored$"),
-        TestIgnored(
-          "only and ignored(weaver.junit.Meta$OnlyFailsOnCiEvenIfIgnored$)"),
```

Comment:
This logic was previously incorrect - we'd previously get a `TestIgnored` notification, as well as a `TestStarted`, `TestFailed`, and `TestFinished` notification.

We now only get `TestStarted`, `TestFailed`, and `TestFinished`.

Given that this is a CI test, and the JUnit runner is unlikely to be used in CI, I doubt this affected anyone.